src/Discovery/test/Eureka.Test/EurekaPostConfigurerTest.cs

```diff
@@ -888,6 +889,57 @@ public void UpdateConfiguration_WithVCAPEnvVariables_AppName_Overrides_VCAPBindi
         Assert.Equal(EurekaPostConfigurer.UnknownZone, map[EurekaPostConfigurer.Zone]);
     }
 
+    [Fact]
+    public void UpdateConfiguration_WithVCAPEnvVariables_ButNoUri_DoesNotThrow()
+    {
+        var vcap_application = @"
+                {
+                    ""application_name"": ""foo"",
+                    ""application_uris"": [ ],
+                    ""name"": ""foo"",
+                    ""uris"": [ ],
+                    ""application_id"": ""ac923014"",
+                    ""instance_id"": ""instance_id""
+                }";
+        var vcap_services = @"
+                {
+                    ""p-service-registry"": [{
+                        ""credentials"": {
+                            ""uri"": ""https://eureka.apps.testcloud.com"",
+                        },
+                        ""label"": ""p-service-registry"",
+                        ""name"": ""myDiscoveryService"",
+                        ""tags"": [
+                            ""eureka""
+                        ]
+                    }]
+                }";
+
+        Environment.SetEnvironmentVariable("VCAP_APPLICATION", vcap_application);
+        Environment.SetEnvironmentVariable("VCAP_SERVICES", vcap_services);
+        Environment.SetEnvironmentVariable("CF_INSTANCE_INDEX", "1");
+        Environment.SetEnvironmentVariable("CF_INSTANCE_GUID", "ac923014-93a5-4aee-b934-a043b241868b");
+        using var sandbox = new Sandbox();
+        var config = new ConfigurationBuilder().AddCloudFoundry().Build();
+        var si = config.GetServiceInfos<EurekaServiceInfo>().First();
+
+        var clientOpts = new EurekaClientOptions();
+        EurekaPostConfigurer.UpdateConfiguration(config, si, clientOpts);
+
+        var instOpts = new EurekaInstanceOptions();
```

Comment:
Abbreviations like 'opts' and 'inst show up in Resharper's spell checker as spelling errors. While they don't break the build, it would be nice if we can try to avoid them.

Reply:
Thanks for the update. However, 'clientOpts' is still there. Did you forget to push?